doc/installing-a-development-environment.md

```diff
@@ -1,5 +1,14 @@
 ## Installing
 
+### Introduction to Installation
```

Comment:
@gandhirajan what is the purpose of this section?

Reply:
@dblotsky Its a kind of heads up to the developer about what he/she is gonna install for this environment and why it is required. Jan and Myself have agreed that it will give more insight for the developer as to why he/she is doing it.

Reply:
the headline actually feels pretty redundant - the paragraph can just sit under `Installing` imo.

Reply:
@janpio Taken care Jan.